### ISSUE TITLE
Strange logging error & lack of response when making a PUT from an angular.js app

### ISSUE THREAD

**Issue by avoidwork** [avoidwork/turtle.io#341]
![screen shot 2015-06-09 at 1 42 27 pm](https://cloud.githubusercontent.com/assets/369698/8065154/9f9e2a5e-0ead-11e5-8d5e-891c67fd9d50.png)

I'm not sure what's going on at the bottom of the screenshot, but the angular app never got the response it expected / it may have been malformed.

**Comment by avoidwork** [comment]
The server has the following route defined:

```javascript
server.put("/", function (req, res) {
	console.log(req.body);
	res.respond("Success");
});
```

So, this is probably a debugging effort with this particular angular app to see what the input is, and why it's being mishandled.

**Comment by avoidwork** [comment]
This has been addressed in a previous release, potentially `4.0.8`, however there is a logging gap from `res` short cuts now that logging is a 'final' step of the pipeline & these shortcuts skip said pipeline.
